### ISSUE TITLE
Synonym Lookup

### ISSUE THREAD

**Issue by rafaelroseira** [datacleaner/DataCleaner#1906]
Does "Synonym Lookup" catalog can be used only with words, and not with expressions? When using a catalog entry such as "bird,yellow bird,blue bird", for instance, "yellow bird" and "blue bird" are not changed for "bird". When using only-one-word catalog, such as "home,house,apartment" for instance, everything works fine.

**Comment by kaspersorensen** [comment]
I do believe that that should work. I recall writing some tests to assert that at least. But it's been a long time. Have you tested this claim?

**Comment by rafaelroseira** [comment]
Yes, I tested a lot. Even using expressions between "", '', `` and with comma at the end of the line.
This is part of my catalog:

Vinícola Aurora,Aurora
Álvaro Palacios,'Alvaro Palacios'
Bacalhôa,"Bacalhôa Vinhos de Portugal"
Bodega Finca Las Moras,"Finca Las Moras",'Finca Las Moras',`Finca Las Moras`,Finca Las Moras
Celaya,Bodegas Celaya,Bodega Celaya
Bodega Norton,Norton

Only lines 1 and 6 work, because the synonym is a simple word.
